packages/hardhat/test/integration-tests/strategy/crab-hedging-time-based.ts

```diff
@@ -278,7 +278,7 @@ describe("Crab flashswap integration test: time based hedging", function () {
 
       await expect(
         crabStrategy.connect(depositor).timeHedge(isSellAuction, expectedAuctionWSqueethEthPrice, {value: expectedEthProceeds.add(1)})
-      ).to.be.revertedWith("can not execute hedging trade");
```

Comment:
Confused why this was passing, is this the weird bug where if part of a revert statement matches it passes?

Reply:
This wasn't ever a revert statement, but was the beginning of "can not execute hedging trade as auction type changed"

Reply:
Nice catch! probably a bug 😅

Reply:
yeah you only need part of the revert message to match to pass